crates/router/src/scheduler/consumer.rs

```diff
@@ -233,12 +234,12 @@ pub async fn run_executor<'a>(
 }
 
 #[instrument(skip_all)]
-pub async fn some_error_handler<E: fmt::Display>(
+pub async fn consumer_error_handler<E: fmt::Display + fmt::Debug>(
     state: &AppState,
     process: storage::ProcessTracker,
     error: E,
 ) -> CustomResult<(), errors::ProcessTrackerError> {
-    logger::error!(pt.name = ?process.name, pt.id = %process.id, %error, "Failed while executing workflow");
+    logger::error!(pt.name = ?process.name, pt.id = %process.id, ?error, "ERROR: Failed while executing workflow");
```

Comment:
Can you add a example log hear, the  log should be proper formatted json. With this `ERROR: Failed while executing workflow`. I don't think so it is a proper json.

Reply:
Sure